src/dictionary/ui/display/WordInformationDisplay.test.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactNode } from 'react'
+import {
+  render,
+  screen,
+  waitForElementToBeRemoved,
+} from '@testing-library/react'
+import { factory } from 'factory-girl'
+import { MemoryRouter, Route, RouteComponentProps } from 'react-router-dom'
+import SessionContext from 'auth/SessionContext'
+import WordInformationDisplay from 'dictionary/ui/display/WordInformationDisplay'
+import WordService from 'dictionary/application/WordService'
+import MemorySession from 'auth/Session'
+import Bluebird from 'bluebird'
+
+jest.mock('dictionary/application/WordService')
+const wordService = new (WordService as jest.Mock<jest.Mocked<WordService>>)()
+
+const session = new MemorySession(['read:words'])
+let word
+
+describe('Fetch word', () => {
+  beforeEach(async () => {
+    word = await factory.build('word')
+    wordService.find.mockReturnValue(Bluebird.resolve(word))
+    await renderWordInformationDisplay()
+    expect(wordService.find).toBeCalledWith('id')
+  })
+  it('Word parts are displayed', async () => {
+    word.lemma.map((lemma) => expectToBeVisible(new RegExp(lemma)))
+    expectToBeVisible(new RegExp(word.guideWord))
+    expectToBeVisible(new RegExp(word.meaning))
+
+    function expectToBeVisible(text: string | RegExp): void {
+      screen
+        .getAllByText(text)
+        .map((gettedText) => expect(gettedText).toBeVisible())
```

Comment:
What's the point of using `map` when you do not use the return value?

Reply:
i am iterating through the list ? alternative would be using a for loop. `expectToBeVisible `is just void

Reply:
`map` is for mapping. `forEach` is for plain iteration.